lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -242,6 +241,11 @@ private JourneyResponse getJourneyResponse(
                 return profileMatchResponse.get();
             }
 
+            // No profile matched but has a pending F2F request
+            if (isF2FIncomplete) {
+                return buildF2FIncompleteResponse(f2fRequest);
+            }
```

Comment:
Can be clubbed with below return ;
return isF2FComplete
                    ? buildF2FNoMatchResponse(areGpg45VcsCorrelated, auditEventUser)
                    : isF2FIncomplete ? buildF2FIncompleteResponse(f2fRequest): buildNoMatchResponse(vcStoreItems, auditEventUser);

Reply:
It's possible, but if feels a bit less readable to me